### PR TITLE
Add a new recipe for the radiant program

### PR DESCRIPTION
**Pull request by Carsten-Ke**
Radiant is a program to annotate protein sequences rapidly with pfam domains.

### CODE REVIEW

**Comment by Carsten-Ke** [comment]
@BiocondaBot please add label

File: recipes/radiant/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - zlib
+    - openmp
+    - boost-cpp >=1.77
+test:
+  commands:
+    - radiant -h
+
+about:
+  home: https://domainworld.uni-muenster.de/data/radiant-db/index.html
+  license: GPLv3+
```
Comment (bgruening) [inline recipes/radiant/meta.yaml:42]: please use SPDX identifiers here

File: recipes/radiant/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  host:
+    - sqlite
+    - bzip2
+    - zlib
+    - openmp
+    - boost-cpp >=1.77
+  run:
+    - sqlite
+    - bzip2
+    - zlib
```
Comment (bgruening) [inline recipes/radiant/meta.yaml:33]:
```suggestion
```

File: recipes/radiant/meta.yaml (hunk 1 above)
Comment (Carsten-Ke) [reply recipes/radiant/meta.yaml]: I changed GPLv3+ to GPL-3.0-or-later

File: recipes/radiant/meta.yaml (hunk 2 above)
Comment (Carsten-Ke) [reply recipes/radiant/meta.yaml]: I removed both requirements

**Review by bgruening — APPROVED** [review]

**Pull request merged**